README.md

```
@@ -0,0 +1,36 @@
+
+---
+
+## ⚙️ Installation
+
+```bash
+composer require --dev haspadar/phpstan-eo-rules
+```
+
+Enable in your `phpstan.neon`:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)